packages/design-system/src/components/ButtonGroup/ButtonGroup.md

```diff
@@ -22,22 +22,29 @@ By default ButtonGroup use internal state to track currently selected button. It
 
 ### Controlled ButtonGroup
 
-If you want to use ButtonGroups as controlled component you can set `currentIndex` props to set which button is active.
+If you want to use ButtonGroups as controlled component you can set `currentindex` props to set which button is active.
 
 ```js
-<ButtonGroup currentIndex={1}>
+initialState = { currentindex: 0 };
+
+handleClick = (index, e) => {
+  console.log('index', index)
```

Comment:
this `console.log` probably should not be here

Reply:
removed at b1005187a0d51f7e88f2f764c0701d21ca564965